transmogrifier/cli.py

```diff
@@ -23,10 +23,16 @@
     help="Filepath for harvested input records to transform",
 )
 @click.option(
-    "-o",
     "--output-file",
-    required=True,
-    help="Filepath to write output TIMDEX JSON records to",
+    required=False,
+    help="Filepath to write output TIMDEX JSON records to. NOTE: this option will be "
+    "removed when output to parquet is finalized.",
+)
+@click.option(
+    "-o",
```

Comment:
We might reconsider the short options (`-o`) altogether, I get the convenience if this is an app that staff are using manually where the input changes all the time but since this primarily triggered by StepFunctions, if we dropped them, we would avoid any future clashes like this for `-o`

Reply:
It's worth considering for sure.  Flipside, our discussion in slack is really reinforcing for me that once we fully commit to "v2" (parquet writing) that `-o` would still be a good and meaningful alias for `--output-location`. 

But agree, if there is _any_ feature flagging going on, it makes it trickier.